docs/content-crag/concepts/solids-pipelines/pipelines.mdx

```diff
@@ -1,196 +1,135 @@
 ---
-title: Pipelines | Dagster
-description: A pipeline is a set of solids with explicit data dependencies on each other, creating a directed acyclic graph, or DAG.
+title: Graphs | Dagster
+description: A graph is a set of ops or sub-graphs with explicit data dependencies on each other.
 ---
 
-# Pipeline
+# Graphs
 
-A pipeline is a set of solids with explicit data dependencies on each other, creating a directed acyclic graph, or DAG.
+A graph is a set of ops or sub-graphs with explicit data dependencies on each other.
```

Comment:
spitballing: thoughts on introducing the notion of "nodes" here, as in "A graph is a set of nodes with explicit data dependencies on each other" and later in the Overview section we expand what a "node" can be (e.g. op, sub-graph which could link to the composite solids page)

Reply:
So far, we've been able to mostly get by without using the word "node", and I think that's a good thing.  It might make sense to add it in the future or farther down the page, but trying to limit the number of concepts for now, at least in the graph introduction.